src/index.js

```diff
@@ -69,7 +69,7 @@ module.exports = (oas, operation, values, auth, lang, oasUrl, harOverride) => {
       }
     }
 
-    targetOpts.apiDefinition = oas;
+    targetOpts.apiDefinition = oas ? oas.getDefinition() : null;
```

Comment:
`httpsnippet-client-target` wants the API definition because it dumps it into its own instance of `oas`.